Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -2007,7 +2007,6 @@ - (void)onPlusButtonPressed
     [actionSheet popoverPresentationController].sourceView = plusButtonImageView;
     [actionSheet popoverPresentationController].sourceRect = plusButtonImageView.bounds;
     
-    [actionSheet mxk_setAccessibilityIdentifier:@"RecentsVCCreateRoomAlert"];
```

Comment:
This line caused VoiceOver to verbally read the actions as "RecentsVCCreateRoomAlert Start Chat" and it doesn't seem to be used anywhere so I removed it.